framework/db/transaction_manager.py

```diff
@@ -427,9 +400,7 @@ def SearchByRegexNames(self, name_list, stats=False, target_id=None):
         + transaction_ids - list of one transaction id per unique match
         + match_percent
         """
-        results = [
-            self.SearchByRegexName(regex_name, stats=stats, target_id=target_id)
-            for regex_name in name_list]
+        results = [self.SearchByRegexName(regex_name, stats=stats, target_id=target_id) for regex_name in name_list]
         return (results)
 
```

Comment:
https://github.com/hTrap/owtf/blob/pep8-fixes/framework/db/transaction_manager.py#L363,
https://github.com/hTrap/owtf/blob/pep8-fixes/framework/db/transaction_manager.py#L381,
https://github.com/hTrap/owtf/blob/pep8-fixes/framework/db/transaction_manager.py#L436,
https://github.com/hTrap/owtf/blob/pep8-fixes/framework/db/transaction_manager.py#L473 - why not like this?

```
        grep_outputs = self.db.session.query(models.GrepOutput.output).filter_by(
            name=regex_name, target_id=target_id).group_by(models.GrepOutput.output).all()
```